src/content/docs/release-notes/docs-release-notes/docs-4-26-2024.mdx

```diff
@@ -0,0 +1,63 @@
+* Updated our NerdGraph synthetics tutorial with a new example [query to retrieve the status of all runtime upgrade tests](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial/#query-runtime-upgrade-status).
+* Our [IBM MQ integration](/install/ibm-mq/) is compatible with IBM MQ version 9.0 or higher and our docs reflect that now.
+* Added manual and automatic instrumentation instructions for [Ruby Lambda functions](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-serverless-monitoring-aws-lambda-layerless/#ruby).
+* In our [OpenTelemetry Java tutorial](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java/#traces), clarified that the OpenTelemetry and New Relic Java agents do error handling differently for traces.
+* With version 9.9.0, the Python agent now monitors [runtime metrics for physical memory](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#runtime-metrics). This is enabled by default.
+* Replaced UI path with direct link to [NerdGraph API explorer](/docs/apis/nerdgraph/get-started/nerdgraph-explorer/#set-up) in our UI.
+* Removed exception to our statement that all [CloudWatch metrics](/docs/infrastructure/amazon-integrations/connect/aws-metric-stream) from all AWS services are available.
+* Updated our [Networks](/docs/new-relic-solutions/get-started/networks) doc to update endpoints and clarify who to reach out to before updating them in future.
+* Updated [browser API `onEnd()`](/docs/browser/new-relic-browser/browser-apis/onend) function syntax.
+* For Kubernetes synthetic monitoring [containerized private minions](/docs/synthetics/synthetic-monitoring/private-locations/install-containerized-private-minions-cpms), removed Windows as a supported deployment option and clarified that Linux containers can only run on Linux nodes.
+* Added a notice on relevant docs about the October 22, 2024 EOL for containerized private minion and its legacy synthetics runtime versions.
+* Various updates and fixes to our [session replay](/docs/browser/browser-monitoring/browser-pro-features/session-replay) docs.
+* When the [Java agent `trim_stats`](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#trim_stats) setting is `true`, a metric is collected when the exclusive time of a segment is less than 2% of the total transaction time.
+* Updated [Java agent classloading issues from JBoss and Wildfly](/docs/apm/agents/java-agent/troubleshooting/classloading-issues-from-jboss-and-wildfly) troubleshooting doc to remove tabs.
+* For [Alerts webhook notification](/docs/alerts-applied-intelligence/notifications/notification-integrations/#set-webhook-destination) integrations, we no longer support OAuth 2.0.
+
+### Release notes and What's New posts
```

Comment:
Order here seems a bit odd, I'd suggest instead:

* All the WN posts, in alphabetical (or chronological?) order
* All the other release notes, in alphabetical order